exporters/zipkin/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020, OpenTelemetry Authors
```

Comment:
Do we need update this to `Copyright 2021`?

Reply:
fixed